.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
+  - 2.0
```

Comment:
Jekyll 3.6 and beyond requires at least Ruby 2.1.0.. technically Jekyll 3.5.x too because Liquid 4.0 requires at least Ruby 2.1

Reply:
This PR itself introduced a development dependency to Jekyll, so I want to make sure it doesn't impact [all other projects using mercenary](https://github.com/jekyll/mercenary/network/dependents)

Reply:
I get that.. Travis wont go :green_heart: because of this discrepancy..

Reply:
Since Mercenary isn't directly related to or dependent on Jekyll, you can simply copy jekyll's `.rubocop.yml` and maintain that here.. that's the best solution IMO..